community/blender/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-musl-fixes.patch
+patch -p1 < 0002-fix-linking-issue.patch
+patch -p1 < 0003-increase-thread-stack-size-for-musl.patch
+
+export CFLAGS="$CFLAGS -D__MUSL__"
+export CXXFLAGS="$CXXFLAGS -D__MUSL__"
+
+cmake -B build \
+    -Cbuild_files/cmake/config/blender_release.cmake \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DWITH_CYCLES_EMBREE=ON \
+    -DWITH_PYTHON_INSTALL=OFF \
+    -DWITH_INSTALL_PORTABLE=OFF \
+    -DWITH_SYSTEM_LZO=ON \
+    -DPYTHON_VERSION="$(python3 -c 'import sys; print("%i.%i" % (sys.version_info.major, sys.version_info.minor))')"
+
+             cmake --build   build
+DESTDIR="$1" cmake --install build
```

Comment:
Export `DESTDIR="$1"` at the top of the buildfile according to https://k1ss.org/wiki/kiss/style-guide#0800

Reply:
In the case of the packages where I put them in front of the cmake command, this is because putting it at the top made the build fail. I suspect it would install some internally shipped buildfiles to `DESTDIR` rather than some relative location.

Reply:
I think the intention i to add at the top `export DESTDIR="$1"`.
By ust adding `DESTDIR="$1"` the variable is not available to cmake lines below.

Reply:
@onodera-punpun 
This is what I meant, I'm sorry I wasn't clearer.

Reply:
@onodera-punpun 
This is what I meant, apologies for not being clearer.

Reply:
The thing is that when putting `export DESTDIR="$1"` at the top (which I do with most packages in this PR, except for `blender` and `openimageio` I think) it breaks the build.

Reply:
That's odd, could you provide a build log.